### ISSUE TITLE
Missing CREDENTIALS env vars for Natural Language

### ISSUE THREAD

**Issue by blowmage** [GoogleCloudPlatform/google-cloud-ruby#1830]
We [recently](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/0d55eb8393325de676eb429fdcae8ea9fc18807b/google-cloud-language/lib/google/cloud/language/credentials.rb#L42-L51) added support for environment variable such as `LANGUAGE_CREDENTIALS`, but these new vars are now [missing](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/f5cc6623400f90bcd37ee1f49c2a7e2f411900f2/google-cloud-language/lib/google/cloud/language/credentials.rb#L24-L25). They should be restored.

**Comment by blowmage** [comment]
@geigerj Is this something that should be addressed in the GAPIC generator configuration?

**Comment by blowmage** [comment]
@jbolinger Is this something you can help with?

**Comment by jbolinger** [comment]
@blowmage sure - looks like this is simply adding the vars that end in `_CREDENTIALS`? Do you know if there was a reason for removing it (@landrito do you know?)

**Comment by jbolinger** [comment]
I see. I can certainly add it to the generator. When you said "generator configuration" before did you mean that this needs some kind of flag or setting vs. always generating it with the additional var names? Does it ever vary by API?

If we don't need additional configuration I can probably make the generator change today but, in any event, we don't need to wait for that if you'd like to resolve this issue now. If this has happened for other APIs it might be useful to update them all at once. Happy to help with that as well if you'd like.

**Comment by blowmage** [comment]
I don't know what is configuration vs. what is non-configuration. (I don't know how the generator works.)

**Comment by blowmage** [comment]
Yes, I believe we would like this applied to the other libraries as well.

**Comment by jbolinger** [comment]
@blowmage I've added it to the generator.

**Comment by blowmage** [comment]
@dazuma What does it take to kick off the source code generation for a gem? I believe updating the generated source code for Natural Language will resolve this issue.

**Comment by dazuma** [comment]
Not much. I can do it.
